.github/workflows/dev.yml

```diff
@@ -25,3 +25,31 @@ jobs:
       - run: pip install -r ./requirements.txt
       - run: pip install .
       - run: python ./agoradatatools/process.py test_config.yaml --authtoken ${{secrets.SYNAPSE_PAT}}
+
+  test:
```

Comment:
Last thing.  I would put this prior to the integration test, and add something like [this](https://github.com/Sage-Bionetworks/Genie/blob/57b6b04a759f5d2198873bc83d2e329ec562453e/.github/workflows/ci.yml#L58) so that the integration test doesn't run if the unit tests don't pass.